JavaScriptCodeConventions.md

```diff
@@ -0,0 +1,58 @@
+# JavaScript Code Conventions
+> NOTE: These conventions also apply to super-sets of JavaScript which are used within Ultimaker such as TypeScript.
+
+## Introduction
+These JavaScript style guidelines are intended to be used by both the R&D and marketing team as part of a broader effort to standardize JavaScript style across the company and as an early facilitator for the fact that Ultimaker continues to offer more web-based products such as the Cura Cloud services and Cura Connect.
+
+## AirBnB as Baseline Style
+To reduce the need to build comprehensive style guidelines ourselves, we have decided to adopt the well-known and widely used **AirBnB style**. The full documentation for AirBnB style [can be found on GitHub](https://github.com/airbnb/javascript). For the most part, this style is already very similar to what is used by front-end teams at Ultimaker with a few differences, which are outlined below. It was also chosen because of it was designed for usage with React.js, which is used by the R&D department. It is applicable to Vue.js as well, however, and can be used by the marketing department.
+
+## Enforcing AirBnB Style in Your Editor
+### Set-Up for VS Code
+1. Install packages globally:
+
+```
+npm i -g eslint eslint-config-airbnb eslint-plugin-import eslint-plugin-jsx-a11y eslint-plugin-react
+```
+2. Install the [`eslint` plugin for VS Code](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint).
+3. Create an `.eslintrc` file in your project’s root directory.
+4. Add the following content to `.eslintrc`:
+
+```
+{
+  "extends": "airbnb",
+  "env": {
+    "node": true,
+    "es6": true,
+    "browser": true
+  },
+  "rules": {
+	  no-underscore-dangle: "allow"
+  }
+}
+```
+
+
+## Required Changes for Adoption
+### Simple to Change
+Some changes can easily be made with a single find & replace operation and a single commit.
+  
+#### Tabs
+- **Existing:** Use 4 spaces for tabs.
+- **AirBnB:** Use 2 spaces for tabs.
+
+#### Strings
+- **Existing:** Use single-quotes.
+- **AirBnB:** Use double-quotes.
```

Comment:
🎉

Reply:
I removed it because I don't think every single change needs to be described and if it's not an exception to AirBnB OR the global style it doesn't really warrant special mention.